engine/src/main/java/org/terasology/rendering/dag/nodes/OpaqueObjectsNode.java

```diff
@@ -16,44 +16,58 @@
 package org.terasology.rendering.dag.nodes;
 
 import org.terasology.engine.ComponentSystemManager;
+import org.terasology.entitySystem.systems.RenderSystem;
 import org.terasology.monitoring.PerformanceMonitor;
 import org.terasology.registry.In;
 import org.terasology.rendering.cameras.Camera;
 import org.terasology.rendering.dag.WireframeCapableNode;
-import org.terasology.rendering.primitives.ChunkMesh;
-import org.terasology.rendering.world.RenderQueuesHelper;
+import org.terasology.rendering.dag.stateChanges.SetViewportToSizeOf;
 import org.terasology.rendering.world.WorldRenderer;
-import org.terasology.rendering.world.WorldRendererImpl;
+
+import static org.terasology.rendering.opengl.DefaultDynamicFBOs.READ_ONLY_GBUFFER;
 
 /**
- * TODO: Diagram of this node
- * Alpha reject is used for semi-transparent billboards, which in turn are used for ground plants.
+ * This node renders the opaque (as opposed to semi-transparent)
+ * objects present in the world. This node -does not- render the landscape.
+ *
+ * Objects to be rendered must be registered as implementing the interface RenderSystem and
+ * take advantage of the RenderSystem.renderOpaque() method, which is called in process().
  */
-public class ChunksAlphaRejectNode extends WireframeCapableNode {
+public class OpaqueObjectsNode extends WireframeCapableNode {
```

Comment:
Github is highlighting this as a rename from ChunksAlphaRejectNode to OpaqueObjectsNode. That's false.